specification/containerinstance/resource-manager/Microsoft.ContainerInstance/stable/2018-06-01/containerInstance.json

```diff
@@ -365,14 +365,14 @@
     },
     "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerInstance/containerGroups/{containerGroupName}/containers/{containerName}/exec": {
       "post": {
-        "operationId": "StartContainer_LaunchExec",
+        "operationId": "Container_Exec",
```

Comment:
can we more specific like `Container_ExecuteCommand`?

Reply:
I'm agree. That would be a good change. Pushing changes now.

Reply:
thank you @samkreter